packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -28,19 +31,24 @@ const EDiscovery = WebexPlugin.extend({
   /**
    * Creates a compliance report with a specific set of search parameters
    * @param {Object} reportRequest - A set of criteria for determining the focus of the search
+   * @param {Object} options - can specify this option :
+   *                           timeoutMs - connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity>} Http response containing the new report record
    */
-  createReport(reportRequest) {
+  createReport(reportRequest, options = {}) {
     if (!reportRequest) {
       throw Error('Undefined parameter');
     }
 
```

Comment:
updated to a named options parameter and using defaults from lodash to set timeoutMs if undefined, i still need to assign a default empty object in the param definition